### ISSUE TITLE
As a system user I want to compare the forecast and in situ data for a single city (Part 2)

### ISSUE THREAD

**Issue by amehta-scottlogic** [ECMWFCode4Earth/vAirify#66]
## Description

This page will be used to compare forecast/in situ for a single city.

This ticket depends on #60  being completed so has scope only for the displaying the in situ data in the UI for a single city

## Acceptance Criteria

- Under route /city/{city_name}, the UI will show the a line chart showing in situ data for a city (next to forecast)
  - Use API /air-pollutant/measurements to fetch forecast data for city (already done in previous ticket)
  - Data here should be aggregated on the front end with AQI levels being calculated. This should be done so that it ties into forecast_valid_time from the forecast (measurements within -/+0.5 hours after each forecast_valid_time is aggregated and averaged together)
  - **The measurement data should tie in with site selection** When sites are selected/deselected, measurements are filtered out accordingly

Test Checklist:
- [x] Test Analysis & discussion with developer
- [x] Write Test Cases / Charters
- [x] Test Cases & Charters Reviewed
- [x] Test Cases Executed
- [x] Test Charters Executed
- [x] Regression tests executed
- [x] Automation Tests added to project

**Comment by mwalker-scottlogic** [comment]
Regression Tests Passed ✅ 

- Backend system tests

![Image](https://github.com/user-attachments/assets/3989ec5a-3dee-46cd-9c35-b5f5e2d8a842)

Frontend system tests

![Image](https://github.com/user-attachments/assets/2d2b3065-6430-4c8b-b495-3dd67166e5e2)

# #

[Playwright Regression 25.07.24.pdf](https://github.com/user-attachments/files/16379307/Playwright.Regression.25.07.24.pdf)

**Comment by mnyamunda-scottlogic** [comment]
# Test analysis
